annotationprocessor/build.gradle

```diff
@@ -7,9 +7,6 @@ buildscript {
     repositories{
         jcenter()
     }
-    dependencies {
```

Comment:
This might be needed for the code generation phase. but I have no idea.
Test things out with the example project. Change some variables around and see if the auto generation works.